docs/SUMMARY.md

```diff
@@ -0,0 +1,34 @@
+* [Home](index.md)
+* [Installation](installation.md)
+* Tutorial
+    * [Introduction to Ibis](tutorial/01-Introduction-to-Ibis.ipynb)
+    * [Aggregating and Joining](tutorial/02-Aggregates-Joins.ipynb)
+    * [Lazy Mode and Logging](tutorial/03-Expressions-Lazy-Mode-Logging.ipynb)
+    * [More Value Expressions](tutorial/04-More-Value-Expressions.ipynb)
+    * [Creating and Inserting External Data](tutorial/05-IO-Create-Insert-External-Data.ipynb)
+    * Advanced Topics
```

Comment:
Do we need to nest it under advanced topics?

Reply:
Probably not.